cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi.help

```diff
@@ -1,8 +1,14 @@
 NAME
+<<<<<<< HEAD
```

Comment:
This conflict should be resolved

Reply:
There are a number of other conflicts in this file too.

Reply:
Resolved the conflicts.